beekeeper-integration-tests/src/test/resources/add_partition.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Hmmm, I guess we're doing all this because Beekeeper currently directly uses the SNS messages. At some time in the future I think we're going to change this to use "DroneFly" downstream of the Kafka events and then implement Beekeeper to look like just a normal HiveMetastoreEventListener. @max-jacobs or others, any ideas how we can try minimise the amount of code that is in this PR that is coupled to SNS while still testing all the paths?

Reply:
@massdosage my understanding of Drone Fly is that, as you say, it will act as a layer downstream of Kafka, but also that it will still land messages on the Apiary SNS topic - in which case we will be using SNS/SQS. How will Beekeeper consume messages from drone fly as a normal HiveMetastoreEventListener?

Reply:
Do you mean that Beekeeper will eventually be pointed directly at Kafka? In which case the additions in `com.expediagroup.beekeeper.integration.model` seem to make the integration tests more extendible and if we can use them as a template for Kafka messages/tests then we could keep them and refactor later.

Reply:
Well, DroneFly will probably expose events via the Hive `MetaStoreEventListener` interface - so an idea could be that Beekeeper runs as a listener on this level and deals with the Hive *Event objects directly and doesn't need to know anything about Kafka and there won't be any need for SQS/SNS. We could decide to still use SQS and SNS like you suggest though, just not sure it would be needed (we probably need to think a bit about how to queue up and distribute the work though).

Reply:
Right this makes sense. So the path scheduler no longer runs as a service, but as a listener on the metastore. In which case there will be more to refactor than just these additions. For the time being I think these additions are probably fine as they seem to make testing quite easy/extendible. I'm happy to keep them in the knowledge that they will most likely be removed in the future.

Reply:
OK, sounds good.